packages/odyssey-react/src/components/Banner/Banner.theme.ts

```diff
@@ -33,8 +37,12 @@ export const theme: ThemeReducer = (theme) => ({
   DismissablePaddingInlineEnd: theme.SpaceScale6,
   DismissablePaddingInlineStart: theme.SpaceScale3,
 
-  CautionBackgroundColor: theme.ColorCautionLight,
-  DangerBackgroundColor: theme.ColorDangerLight,
-  InfoBackgroundColor: theme.ColorPrimaryLight,
-  SuccessBackgroundColor: theme.ColorSuccessLight,
+  CautionBackgroundColor: theme.ColorBackgroundCautionLight,
+  CautionIconColor: theme.ColorCautionDark,
+
+  DangerBackgroundColor: theme.ColorBackgroundDangerLight,
+  DangerIconColor: theme.ColorDangerBase,
+
+  InfoBackgroundColor: theme.ColorBackgroundPrimaryLight,
+  InfoIconColor: theme.ColorPrimaryBase,
```

Comment:
Your PR description suggested that we are removing theme variables that were not accessible to be used via props? Is that correct?


> _"Some theme variables are removed; however, the React API does not allow the Success variant, so these were extraneous."_

Reply:
The Success variant isn't available via the API, so I pulled the related token. Looks like it was just that one, since IconColor wasn't available yet.